src/webgl/p5.RendererGL.js

```diff
@@ -631,7 +635,22 @@ p5.RendererGL.prototype._getColorShader = function () {
   return this._defaultColorShader;
 };
 
+p5.RendererGL.prototype.getTexture = function (img) {
+  // efficiency?
+  var f = function(element) {
+    return element.src === img;
+  };
+
+  var tex = this.textures.find(f);
+  if (tex === undefined) {
+    tex = new p5.Texture(this, img);
+    this.textures.push(tex);
+  }
 
+  tex.set(img);
```

Comment:
Hey, I'm not sure I'm understanding these two comments together.

at present, `getTexture` is only used when setting the uniform, so it will create the texture if it can't find an existing one or or just update it if it finds one. Would it be clearer if the method was called `createOrFindTexture` and `set` was called in the shader?

`p5.Shader.bindTextures` is where the texture is actually fetched/bound for rendering (though not updated) but not with `getTexture`—instead it uses the cached texture object associated with shader's sampler uniforms.